### ISSUE TITLE
ArticleViewer open/close buttons unlabeled

### ISSUE THREAD

**Issue by bwreid** [WikiEducationFoundation/WikiEduDashboard#2490]
The ArticleViewer's open and close buttons are currently not accessible in that they have no labels. When used by a screenreader, it is difficult to understand what each button is.

We need to add some kind of label to the ArticleViewer's open and close buttons. I believe the Uploads tab on the course page has some examples of visual buttons that contain accessible text.

**Comment by zainabsiddiqui** [comment]
Hey @bwreid! I'm applying to this project through Outreachy. Could I try this for my first contribution?

I've taken a look at the ArticleViewer JSX file and it seems attaching an aria-label attribute to the buttons would fix accessibility concerns. Am I on the right track?

**Comment by bwreid** [comment]
Hi @zainabsiddiqui. Yes, this could be a good first issue to work on. You are on the right track! [This section](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/app/assets/javascripts/components/nav/nav.jsx#L99-L110) of the nav bar might be a good reference. Additionally, you can download a screen reader for Chrome if you'd like to try it out.

**Comment by zainabsiddiqui** [comment]
@bwreid quick question about the ArticleViewer open button: I downloaded a screenreader and noticed that it reads the name of the article when clicked. Won't adding a label to that override the reading out of the title itself? 

This is not the case with the close button, as that is just an icon.

**Comment by bwreid** [comment]
No worries, @zainabsiddiqui -- Yep, it can be a bit tricky. Let me know if there's anything else I can help with.  :)
